source/_integrations/manual.markdown

```diff
@@ -134,7 +136,7 @@ In the configuration example below:
 # Example configuration.yaml entry
 alarm_control_panel:
   - platform: manual
-    name: Home Alarm
+    name: HA Alarm
```

Comment:
This is an example...

There is no need in changing this. Furthermore, we try not to use `HA` as an abbreviation.